Question9.java

```diff
@@ -0,0 +1,20 @@
+class Solution {
+    public boolean isPalindrome(int x) {
+        ArrayList<Integer> al = new ArrayList<Integer>();
```

Comment:
Try not to use lower case variable names unless it is a temporary variable that doesn't span more than 10 lines. Suggest use `A`. Technically `A` isn't much better, but at least it differentiates itself so you can tell it is significant, i.e. a container, and not a local variable.